components/tools/OmeroWeb/test/integration/test_tree.py

```diff
@@ -1174,7 +1219,7 @@ def tagset_hierarchy_userA_groupA(request, userA, userB,
     image_link.child = TagAnnotationI(tags[0].id.val, False)
     get_connection(userB).getUpdateService().saveObject(image_link)
 
-    # links is: project, dataset, image, screen, plate, acquisition
+    # links is: project, dataset, image, screen, plate, acquisition, well, well
```

Comment:
That's what we're testing here: See ```def expected_wells(user, wells):``` which generates the dictionary that we compare to what's returned by the tagged json, and includes 'name'.